v20.2/performance-benchmarking-with-tpc-c-10k-warehouses.md

```diff
@@ -194,16 +200,16 @@ CockroachDB offers a pre-built `workload` binary for Linux that includes the TPC
     $ wget https://edge-binaries.cockroachdb.com/cockroach/workload.LATEST -O workload; chmod 755 workload
     ~~~
 
-3. Import the TPC-C dataset:
+3. Import the TPC-C dataset: (<strong><font style="size:xx-large" color="red">FIXME</font></strong>: is this still the right command?)
 
     {% include copy-clipboard.html %}
     ~~~ shell
     $ ./workload fixtures import tpcc \
-    --warehouses 10000 \
+    --warehouses 13000 \
```

Comment:
Is this the right command to import this?  I blindly edited to say `13000` but have no idea.